.github/workflows/oclif-release.yml

```diff
@@ -30,10 +30,6 @@ jobs:
             publish: homebrew
     steps:
       - uses: actions/checkout@v2
-      - name: Set up Node.js
-        uses: actions/setup-node@v2
-        with:
-          node-version: '18.x'
```

Comment:
Since this is going to be built in 18.x and others in 16.x, will it make any difference?

Reply:
Runner images has node 18 by default.